docs/rebase.md

```diff
@@ -83,6 +95,10 @@ When running this command, you will be prompted for the branch on top of which y
 
 If any merge conflicts occur while rebasing, the state of the operation will be reflected in the dashboard, with successful rebased commits showing a check mark, and the commit with the current merge conflict showing an X as well as a list of files with conflicts.
 
+#### Toggle perserve merges mode (`m`)
+
+In default, merges will be lost when rebasing. If perserve merges mode is on, commits within a merge are consolidated in a single item. If any conflicts occur while rebasing a merge commit, the commits within the merge will be shown and the conflicting commit will be marked.
```

Comment:
I see, so without the preserving mode I will see all the commits "behind a merge", but I'd lose the merge commit. I guess I'd have to keep using "rebase -i -p" for these rare cases.